keyboards/hotdox/config.h

```diff
@@ -0,0 +1,82 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0xE11D
+
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Z.P.Z.O.
+#define PRODUCT         HotDox
+#define DESCRIPTION     QMK keyboard firmware for HotDox
+
+/* key matrix size */
+#define MATRIX_ROWS 6
+#define MATRIX_COLS 14
+
+#define MOUSEKEY_INTERVAL       20
+#define MOUSEKEY_DELAY          0
+#define MOUSEKEY_TIME_TO_MAX    60
+#define MOUSEKEY_MAX_SPEED      7
+#define MOUSEKEY_WHEEL_DELAY    0
+
+#define TAPPING_TOGGLE  1
+
+/* define if matrix has ghost */
+//#define MATRIX_HAS_GHOST
+
+#define TAPPING_TERM    200
+#define IGNORE_MOD_TAP_INTERRUPT // this makes it possible to do rolling combos (zx) with keys that convert to other keys on hold (z becomes ctrl when you hold it, and when this option isn't enabled, z rapidly followed by x actually sends Ctrl-x. That's bad.)
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+/* key combination for command */
+#define IS_COMMAND() ( \
+    get_mods() == MOD_MASK_CTRL || \
+    get_mods() == MOD_MASK_SHIFT   \
+)
+
+#define BACKLIGHT_PIN B7
+/* number of backlight levels */
+#define BACKLIGHT_LEVELS 3
+
+#define LED_BRIGHTNESS_LO       15
+#define LED_BRIGHTNESS_HI       255
+
+/* fix space cadet rollover issue */
+#define DISABLE_SPACE_CADET_ROLLOVER
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCE    5
+
+
+#define USB_MAX_POWER_CONSUMPTION 500
+
+/* NKRO */
+#ifndef FORCE_NKRO
+    #define FORCE_NKRO  // Depends on NKRO_ENABLE.
```

Comment:
Is there a reason for forcing NKRO on? 

If there is, that's absolutely fine.  But generally, it's not a good idea, as it may prevent the board from being used in the system BIOS on system systems.    It's not too common, but it is a serious concern when having it enabled by default.